homeassistant/bootstrap.py

```diff
@@ -14,6 +14,7 @@
 
 import homeassistant.components as core_components
 from homeassistant.components import persistent_notification
+from homeassistant.components.packages import merge_packages_config
```

Comment:
This is not a component. Components have a `DOMAIN` and a `setup` method and register services or maintain states. This should live in `homeassistant.config`

I'm still not convinced if merging is the right way to go. We're going to make it SUPER complicated for users to figure out why something is happening or why something is wrong. Also, this is another one of those things that will not work with any config UI.

Instead of merging, we should explore:
 - calling `bootstrap.setup_from_config` multiple times with different files
 - multiple HASS instances linked to one another